qiskit_ionq/ionq_client.py

```diff
@@ -84,10 +103,14 @@ def submit_job(self, job) -> dict:
         as_json = qiskit_to_ionq(
             job.circuit, job.backend().name(), job.backend().gateset(), job._passed_args
         )
-        req_path = self.make_path("jobs")
-        res = requests.post(req_path, data=as_json, headers=self.api_headers)
-        if res.status_code != 200:
-            raise exceptions.IonQAPIError.from_response(res)
+        while True:
```

Comment:
Worth figuring out how to use [`retry`](https://pypi.org/project/retry/) here?

Reply:
very yes, I hated writing manual retries, haha.

Will take a look at what some minimal testing looks like with retry as well.